scripts/redirect_url_utils.py

```diff
@@ -25,11 +25,31 @@
         <meta http-equiv="REFRESH" content="0.0;url={redirect_to_url}" />
     </head>
     <body></body>
-</html>'''
+</html>
+'''
 
 def apply_map():
+  _add_redirects_from_file(FILE_TO_URL)
+
+def update_map(file):
+  """Update the main redirects file with the new redirect pairs in 'file'."""
+  if not os.path.isfile(file):
+    print("'%s' is not a file. Try again." % file)
+    return False
+  _add_redirects_from_file(file)
```

Comment:
ah, that is where new redirects are added, then we update the map.

so the next call to `save_map` should pick newly created redirects and the file with mapping will remain the same.

Reply:
ignore this one